src/zlib.rs

```diff
@@ -26,9 +23,9 @@ use crate::checksum;
 use crate::deflate;
 use crate::finish::{Complete, Finish};
 use crate::lz77;
```

Comment:
```suggestion
use crate::lz77;
#[cfg(not(feature = "std"))]
use alloc::{borrow::ToOwned, string::ToString, vec, vec::Vec};
```

Reply:
These are needed in the test module, and are declared unused if put in the top module. Done in the test module.